src/client/Client.cpp

```diff
@@ -33,15 +32,16 @@ void Client::draw(sf::RenderTarget& target) {
 
 void Client::update() {
     // Get group circle updates
-    std::vector<group_circle_update> group_circle_updates = mNetworkingClient->getClientGroupUpdates();
+    std::vector<client_group_update> client_group_updates = mNetworkingClient->getClientGroupUpdates();
     std::vector<int> client_ids;
-    for (const auto group_circle_update : group_circle_updates) {
-        client_ids.push_back(group_circle_update.client_id);
-    }
+    std::transform(
+        client_group_updates.begin(), client_group_updates.end(), std::back_inserter(client_ids),
+        [](client_group_update cgu){return cgu.client_id;}
```

Comment:
what is this a lambda?

Reply:
Given a `client_group_update` struct it returns its `client_id` field.